src/interfaces.ts

```diff
@@ -266,9 +266,14 @@ export interface ICoreKit {
   commitChanges(): Promise<void>;
 
   /**
-   * Export the user's current TSS MPC account as a private key
+   * Export the user's current SECP256k1 TSS MPC account as a private key
```

Comment:
This also works for ed25519. In which case it will output the private key scalar.